src/library/simplified/extension/state-name.js

```diff
@@ -56,6 +56,10 @@ else if (utag.isMCO()) {
         utag_data["state"] = utag_data['entity.checkout.flightOffers.0.destinationAirportCityState'].split(",")[1].trim();
     }
 }
+else if (utag.isDestinationLandingPage() && b['entity.tripDetails.hotelOffer.hotelDescription.destination'] != undefined)
+{
+    b['state'] = b['entity.tripDetails.hotelOffer.hotelDescription.destination'].split(",")[1];
```

Comment:
why using b instead of the original utag data? Also what happens if the destination text has no comma, i guess undefined will be returned, how that impacts the pixel data?